app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsCardsItemFragment.kt

```diff
@@ -28,38 +27,41 @@ import org.wikipedia.util.log.L
 import org.wikipedia.views.ImageZoomHelper
 
 class SuggestedEditsCardsItemFragment : SuggestedEditsItemFragment() {
+    private var _binding: FragmentSuggestedEditsCardsItemBinding? = null
```

Comment:
Should we set this to `null` in `onDestroy()`?

Reply:
Yes. Done.